packages/components/input/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@spark-ui/input",
+  "version": "0.0.0",
+  "description": "Component used to get user input in a text field",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build": "vite build"
+  },
+  "dependencies": {
+    "@spark-ui/form-field": "^0.1.1",
+    "class-variance-authority": "0.5.2"
+  },
+  "peerDependencies": {
```

Comment:
I think we need a peer-deps package for handling the core dependencies easier from only one place, WDYT?

Reply:
Yes, that would be nice 👍🏻